README.md

```diff
@@ -78,6 +89,7 @@ Write your code here...
 
 ```ruby
 # code here
+puts "Belle is friends with #{friends}!"
 ```
```

Comment:
This would require using the .each method.
Answer:
`friends.each do |friend|
  puts "Belle is friends with #{friend}"
end`